### PR TITLE
Added metric cluster free memory

### PR DESCRIPTION
**Pull request by goruha**
## What
* Added metric cluster memory free %
* Added alert on low cluster memory free %

## Why
* Cluster memory measure is very important

### CODE REVIEW

File: incubator/prometheus-alerts/templates/configmap.yaml [hunk 1]
```diff
@@ -42,4 +42,31 @@ data:
...
+
+    ALERT ClusterMemoryLow
+      IF cluster:memory_free:percent <= {{ .Values.alerts.cluster_memory_low.critical }}
+      FOR 30s
+      LABELS {
+        service = "k8s",
+        severity = "critical",
+      }
+      ANNOTATIONS {
+        summary = "Cluster ran out of  memory",
```
Comment (osterman) [inline incubator/prometheus-alerts/templates/configmap.yaml]: extra whitespace `  `

File: incubator/prometheus-alerts/templates/configmap.yaml [hunk 2]
```diff
@@ -42,4 +42,31 @@ data:
...
+        description = "Cluster ran out of  memory",
```
Comment (osterman) [inline incubator/prometheus-alerts/templates/configmap.yaml]: Extra whitespace

File: incubator/prometheus-alerts/templates/configmap.yaml (hunk 1 above)
Comment (goruha) [inline incubator/prometheus-alerts/templates/configmap.yaml]: Done

File: incubator/prometheus-alerts/templates/configmap.yaml (hunk 2 above)
Comment (goruha) [inline incubator/prometheus-alerts/templates/configmap.yaml]: Done

**Pull request merged**